pkg/webui/console/views/device-general-settings/messages.js

```diff
@@ -34,6 +34,7 @@ const messages = defineMessages({
   activationModeUnknown: 'Activation mode unknown because Network Server is not available',
   notInCluster: 'Not registered in this cluster',
   updateSuccess: 'The end device has been updated successfully',
+  keysResetWarning: 'You can not read the end device keys, but you can overwrite it if you want',
```

Comment:
The tone is too casual. I suggest
```suggestion
  keysResetWarning: "You're not authorized to view the end device keys. However, overwriting is allowed.",
```